phpBB/language/en/acp/common.php

```diff
@@ -219,6 +219,7 @@
 	'ACP_VIEW_GLOBAL_MOD_PERMISSIONS'	=> 'View global moderation permissions',
 	'ACP_VIEW_USER_PERMISSIONS'			=> 'View user-based permissions',
 
+	'ACP_WEBPUSH_SETTINGS'		=> 'Webpush settings',
```

Comment:
I see some different ways of writing "Webpush", "WebPush" and "web push". I think the correct way would be "Web Push" but while this is technically correct, maybe just referring to "Push Notifications" would be more clear for our end users. What do you think?